src/components/ReportActionItem/IOUAction.js

```diff
@@ -78,6 +78,8 @@ const IOUAction = (props) => {
         }
     };
 
+    const shouldShowIOUPreview = props.isMostRecentIOUReportAction || props.action.originalMessage.type === 'pay';
```

Comment:
While we could include 'pay' in the array, it doesn't really make sense because it is not a IOURequest. Also, we're going to completely remove this condition with my next PR.

Reply:
We can do that